src/java/org/apache/cassandra/service/snapshot/SnapshotDetailsTabularData.java

```diff
@@ -18,21 +18,27 @@
 package org.apache.cassandra.service.snapshot;
 
 import java.util.Set;
-import javax.management.openmbean.*;
+import javax.management.openmbean.CompositeDataSupport;
+import javax.management.openmbean.CompositeType;
+import javax.management.openmbean.OpenDataException;
+import javax.management.openmbean.OpenType;
+import javax.management.openmbean.SimpleType;
+import javax.management.openmbean.TabularDataSupport;
+import javax.management.openmbean.TabularType;
 
 import org.apache.cassandra.io.util.FileUtils;
 
 public class SnapshotDetailsTabularData
 {
-
     private static final String[] ITEM_NAMES = new String[]{"Snapshot name",
                                                             "Keyspace name",
                                                             "Column family name",
                                                             "True size",
                                                             "Size on disk",
                                                             "Creation time",
                                                             "Expiration time",
-                                                            "Ephemeral"};
+                                                            "Ephemeral",
+                                                            "Raw true size"};
```

Comment:
Do we need the `true` portion of the name? It somewhat makes it look like we have a non true size :-P

Reply:
We indeed do have non-true size, that field is called "Size on disk".

Reply:
Got it. Thanks for the clarification!